scanner/templates/laravel/.fly/nginx-default

```diff
@@ -24,6 +24,7 @@ server {
 	# Pass "*.php" files to PHP-FPM
 	location ~ \.php$ {
 			include snippets/fastcgi-php.conf;
+			fastcgi_param  HTTP_X_FORWARDED_FOR $http_fly_client_ip;
```

Comment:
I know I'm nitpicking, but there's 2 spaces after `fastcgi_param` 👀

Reply:
thanks!

Reply:
Could the spacing in `nginx-default`also be improved in this PR? It looks a bit weird. When I run `fly launch` the first thing I do is fix the weird spacing in `nginx-default` :)) My preference would be:

- Spaces instead of tabs
- 4 spaces for indentation

<img width="597" alt="image" src="https://user-images.githubusercontent.com/9981548/191489958-e0a96eee-9da0-4a2d-b066-ab6795e1f3bc.png">

Reply:
Agree!